grove/tests/ising/test_ising.py

```diff
@@ -0,0 +1,89 @@
+
+import pytest
+from grove.ising.ising_qaoa import ising_qaoa
+from grove.ising.ising_qaoa import QAOA_ising
+from grove.ising.ising_qaoa import ising
+from grove.pyqaoa.utils import compare_progs
+from pyquil.quil import Program
+from pyquil.paulis import PauliTerm, PauliSum
+from pyquil.gates import H, CNOT, RZ
+import numpy as np
+from mock import Mock, patch
+import pyquil.api as qvm_mod
+
+
+def test_pass_hamiltonians():
+    ref_ham = [PauliSum([PauliTerm("X", 0, -1.0)]), PauliSum([PauliTerm("X", 1,
+                                                                        -1.0)])]
+    cost_ham = [PauliTerm("I", 0, 0.5) + PauliTerm("Z", 0, -0.5) *
+                PauliTerm("Z", 1, 1.0)]
+    fakeQVM = Mock()
```

Comment:
This seems like the second test failure ... Maybe we need more spec'ing of the Mock